app/controllers/v0/claims_controller.rb

```diff
@@ -9,6 +9,10 @@ def index
              each_serializer: ClaimSerializer
     end
 
+    def request_decision
+      render json: current_user.request_claim_decision(params[:id])
```

Comment:
I think that would be fine, but if we do that, we should check server side that they sent us success: true. I've gotten success: false for some test uploads.